tests/globalhelper/runhelper.go

```diff
@@ -47,5 +48,44 @@ func LaunchTests(testSuites []string, skipRegEx string) error {
 	cmd.Args = append(cmd.Args, testArgs...)
 	cmd.Dir = Configuration.General.TnfRepoPath
 
+	if os.Getenv("DEBUG_TNF") == "true" {
```

Comment:
Please initiate DEBUG_TNF and LOG_LEVEL in config.go (Config struct) with rest of env vars.

Reply:
Done.